lib/disk_space.rb

```diff
@@ -2,7 +2,7 @@
 
 class DiskSpace
   def self.uploads_used_bytes
-    if Discourse.store.external?
+    if Discourse.store.external? || !SiteSetting.enable_precise_upload_used_bytes?
```

Comment:
I was thinking of something like this instead:

```
  def self.uploads_used_bytes
     Upload.sum(:filesize).to_i + OptimizedImage.sum(:filesize).to_i 
  end
```

If we can trust this calculation for uploads stored on a cloud storage provider, we should be able to do the same for local uploads.

Reply:
Yup 👍

Reply:
Fixed in this PR. Is it the best practice? Or do you prefer creating a new PR? Thanks